app/controllers/patients_controller.rb

```diff
@@ -101,6 +101,23 @@ def find_patient
 
   def patient_params
     params.require(:patient).permit(
+      
+      :name, :primary_phone, :other_contact, :other_phone,
+      :other_contact_relationship, :line, :voicemail_preference, :spanish,
+      # fields in dashboard
+      # :clinic_name,
+      :appointment_date,
+      :age, :race_ethnicity, :city, :state, :county, :zip, :employment_status, :income,
+      :household_size_adults, :household_size_children, :insurance,
+      :referred_by, :initial_call_date, :urgent_flag,
+      :clinic_id,
+      pregnancy: [:last_menstrual_period_days, :last_menstrual_period_weeks,
+                  :resolved_without_dcaf, :referred_to_clinic, :procedure_cost,
+                  :pledge_sent, :patient_contribution, :naf_pledge, :dcaf_soft_pledge],
+      special_circumstances: [],
+      fulfillment: [:fulfilled, :procedure_date, :gestation_at_procedure,
+                    :procedure_cost, :check_number, :check_date]
+
```

Comment:
Looks like the merge got a bit mucked up! These lines (104-120) should be deleted, as all of them are redundant to lines 77-100 + line 121. 

Not too surprising considering how much we changed the object around on you mid ticket :)